cumulus/pallets/parachain-system/src/lib.rs

```diff
@@ -117,6 +114,9 @@ pub use pallet::*;
 
 const LOG_TARGET: &str = "parachain-system";
 
+//TODO: https://github.com/paritytech/polkadot-sdk/issues/9428
+const FIXED_CLAIM_QUEUE_OFFSET: u8 = 0;
```

Comment:
This needs to be equal to the the relay parent offset. But, it should work fine with 0 as long as the core doesn't change. Otherwise, when building on an older relay parent, the offset would determine some core assignment from the past which is no longer assigned to the para in the next RC block.